security-jwt/src/main/java/io/micronaut/security/token/jwt/cookie/JwtCookieClearerLogoutHandler.java

```diff
@@ -39,46 +40,76 @@
 @Singleton
 public class JwtCookieClearerLogoutHandler implements LogoutHandler {
 
+    @Deprecated
     protected final JwtCookieConfiguration jwtCookieConfiguration;
     protected final String logout;
+    protected final AccessTokenCookieConfiguration accessTokenCookieConfiguration;
+    protected final RefreshTokenCookieConfiguration refreshTokenCookieConfiguration;
 
     /**
      * @param jwtCookieConfiguration JWT Cookie Configuration
      * @deprecated Use {@link JwtCookieClearerLogoutHandler#JwtCookieClearerLogoutHandler(JwtCookieConfiguration, RedirectConfiguration)} instead.
      */
     @Deprecated
     public JwtCookieClearerLogoutHandler(JwtCookieConfiguration jwtCookieConfiguration) {
-            this.jwtCookieConfiguration = jwtCookieConfiguration;
-            this.logout = jwtCookieConfiguration.getLogoutTargetUrl();
+        this.jwtCookieConfiguration = jwtCookieConfiguration;
+        this.accessTokenCookieConfiguration = jwtCookieConfiguration;
+        this.refreshTokenCookieConfiguration = null;
+        this.logout = jwtCookieConfiguration.getLogoutTargetUrl();
     }
 
     /**
      * @param jwtCookieConfiguration JWT Cookie Configuration
      * @param redirectConfiguration Redirect configuration
      */
-    @Inject
+    @Deprecated
     public JwtCookieClearerLogoutHandler(JwtCookieConfiguration jwtCookieConfiguration,
                                          RedirectConfiguration redirectConfiguration) {
         this.jwtCookieConfiguration = jwtCookieConfiguration;
+        this.accessTokenCookieConfiguration = jwtCookieConfiguration;
+        this.refreshTokenCookieConfiguration = null;
```

Comment:
Should it be better to set 


`this.refreshTokenCookieConfiguration = jwtCookieConfiguration;`
instead of: 

`this.refreshTokenCookieConfiguration = null;`